lib/ChangeTypes.js

```diff
@@ -84,6 +84,10 @@ module.exports = {
   FIND_GCD: 'FIND_GCD',
   // e.g. (5*3)/(2*3) -> 5/2
   CANCEL_GCD: 'CANCEL_GCD',
+  // e.g. 1 2/3 -> 5/3
+  CONVERT_MIXED_NUMBER_TO_IMPROPER_FRACTION: 'CONVERT_MIXED_NUMBER_TO_IMPROPER_FRACTION',
+  // e.g. 1 2/3 -> ((1 * 3) + 2) / 3
+  IMPROPER_FRACTION_NUMERATOR: 'IMPROPER_FRACTION_NUMERATOR',
```

Comment:
I'm confused about what it's doing so I can't really name it anything :p can you ask Gen how she'd describe it?